org.kde.PlatformTheme.QAdwaitaPlatform.json

```diff
@@ -0,0 +1,67 @@
+{
+  "id": "org.kde.PlatformTheme.QAdwaitaPlatform",
```

Comment:
No. I published flatpak for the first time, and because this project is fork of qgnomeflatform, I've taken json from their repo. If there is a problem with the domain, i can replace it with mine.

Reply:
If this is not a KDE project then this should not use this namespace

Reply:
@JakobDev  `org.kde.PlatformTheme` is an extension point

https://invent.kde.org/packaging/flatpak-kde-runtime/-/blob/qt5.15lts-22.08/org.kde.Sdk.json.in#L46

Reply:
So i need to keep this domain or change to mine?

Reply:
Please keep it as is.